src/pass/verify_memory.cc

```diff
@@ -139,7 +139,7 @@ class MemoryAccessVerifier final : protected IRVisitor {
 
   /// Check if a given DLDeviceType/TVMDeviceExtType value denotes GPU device.
   static bool IsGPUDevice(int dev_type) {
```

Comment:
Thanks for your comment.  I've introduced IsFPGADevice() instead of modifying IsGPUDevice().  Now, we can check that the memory access is not to host with !(IsGPUDevice() || IsFPGADevice).